ops/db-connect.sh

```diff
@@ -1,30 +1,138 @@
-#!/bin/bash
-
-# Call this to connect to cloud Postgres for a specific Data Repo instance.
-# For example, to connect to the dev datarepo database as user mm, run:
-# $ DB=datarepo SUFFIX=mm ENVIRONMENT=dev ./db-connect.sh
-# To connect to a user instance's stairway database, run:
-# $ DB=stairway-mm SUFFIX=mm ENVIRONMENT=dev ./db-connect.sh
+#!/bin/sh
+#
+# Connect to a Terra Data Repo database in an environment.
+#
+# It uses the gcloud CLI to get the credentials for the cluster, kubectl to port
+# forward to the SQL proxy pod, and psql to connect to the database.
 #
-# If the connection times out, check that you are on the Broad VPN and are connected to correct
-# Kubernetes cluster.
+# You MUST have gcloud, kubectl, jq, and psql installed to run this script. In
+# addition, you MUST be connected to the VPN to access the database.
 #
-# If you get an AlreadyExists error, it's possible the pod didn't shut down properly. You can
-# delete it using kubectl, where ZZ is the SUFFIX.
-# $ kubectl --namespace ZZ delete pod ZZ-psql
+# See usage section below for more details. All arguments are optional.
+#
+
+set -eu
+
+usage() {
+  cat <<EOF
+Usage: $0 [OPTION]...
+Connect to a Terra Data Repo database in an environment.
+
+You MUST have gcloud, kubectl, jq, and psql installed to run this script. In
+addition, you MUST be connected to the VPN to access the database.
+
+  --env ENV             Environment to connect to (default: dev)
```

Comment:
I think it would be nice to know that at present, only dev and staging are supported by this script.